django_freeradius/static/django-freeradius/js/radcheck.js

```diff
@@ -1,8 +1,11 @@
+var django = null;
+
 django.jQuery(function ($) {
     'use strict';
     // stop here if in add mode
+    var gettext = false;
```

Comment:
`gettext` is an internationalization utility defined by django, if you do this you will effectively delete it.

Add this to the `predef` globals instead, remove this line.